### PR TITLE
Add method to trigger low memory notification

### PR DESCRIPTION
**Pull request by byronpatrick22**
Force v8 garbage collection in context object deallocation

### CODE REVIEW

**Comment by tbodt** [comment]
I don't think this is a good idea since "low on memory" is a lie. Most programs should let V8 collect its garbage when it wants. If your program is different, we can add a low memory notification method to v8py, but it shouldn't be called on every context destroy.

**Comment by tbodt** [comment]
Seems good but I don't think this should have a test. "Low memory" is a hint, not a requirement that something should happen.

File: v8py/convert.cpp [hunk 1]
```diff
@@ -201,9 +201,8 @@ Local<Value> js_from_py(PyObject *value, Local<Context> context) {
     }
 
     if (PyList_Check(value) || PyTuple_Check(value)) {
-        int length = PySequence_Length(value);
         Local<Array> array = Array::New(isolate, length);
```
Comment (byoung0589) [inline v8py/convert.cpp]: Hey @tbodt while building and testing this out, I noticed that  length was not replaced with PySequence_Length(value) , seemed too small to open a PR for.

Comment (tbodt) [reply v8py/convert.cpp]: fuck

**Pull request merged**